front_end_pocket_poll/lib/events_widgets/events_list.dart

```diff
@@ -123,36 +131,67 @@ class _EventsListState extends State<EventsList> {
         ],
       );
     } else {
-      // sort the events
+      //sort the events
```

Comment:
So good job spotting that error. But I'm wondering if you could move this sort to that Sorter class i made a while back? Just because this is such a long method now that kind of overwhelms the rest of the logic in this file. You would just pass in the eventcard list by reference and sort it.

Reply:
Totally makes sense. Moved the sort method out to the Sorter class.